test/units/TEST-87-AUX-UTILS-VM.bootctl.sh

```diff
@@ -13,6 +13,11 @@ if [[ ! -d /usr/lib/systemd/boot/efi ]]; then
     exit 0
 fi
 
+if [[ -v UBSAN_OPTIONS ]] ; then
+    echo "UBSAN_OPTIONS is set, skipping bootctl tests that trigger false positives"
```

Comment:
No... Please do not hide the issue.
The issue should be fixed by #38386.
Feel free to pick the commit into this PR.

Reply:
Already tried, doesn't work, it's a false positive. It's not really important, what matters more is having functional coverage of the features paths, given how often regressions happen